### PR TITLE
Add registration page

### PR DESCRIPTION
**Pull request by Omams21**

### CODE REVIEW

**Review by mykelmatics — COMMENTED** [review]
It really looks good, I just dropped  few comments

File: Desktop/property-pro-app/src/pages/Signin/Signin/Signin.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+import "./signin.css";
+
+export default function SignIn() {
+  return (
+    <div className="signinwrap">
+      <div className="companyLogo">
+        <i class="fa fa-home" aria-hidden="true"></i>
+        <span className="name">PropertyPro</span>
+      </div>
+      <div className=" firstcontainer">
```
Comment (Shukura2) [inline Desktop/property-pro-app/src/pages/Signin/Signin/Signin.js:11]: Same here, try to look through the code and make similar changes where needed.

File: Desktop/property-pro-app/src/pages/Signin/Signin/Signin.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        />
+        <button class="signin-btn ">Sign In</button>
+
+        <p className="password-button"> Forgot Your Password?</p>
+      </div>
+      <hr></hr>
+      <div className="secondcontainer">
+        <p className="connect"> Or Connect With : </p>
+        <div className="apple">
+          {" "}
```
Comment (Shukura2) [inline Desktop/property-pro-app/src/pages/Signin/Signin/Signin.js:39]: Let's take this out, since its not part of the code.

**Review by Shukura2 — CHANGES_REQUESTED** [review]
Weldone.

**Pull request merged**